src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule332nolinewrap/InputNoLineWrapGood.java

```diff
@@ -3,7 +3,6 @@
 import static java.math.BigInteger.ZERO; //ok
 
 import com.puppycrawl.tools.checkstyle.checks.design.FinalClassCheck; //ok
-
 import java.util.concurrent.atomic.AtomicIntegerFieldUpdater; //ok
```

Comment:
Input for NoLineWrap should check line wraps only.

Reply:
it is better to revert this Input change.
Such Input should not participate in CustomImportOrder, so it can be in any form. Sometime we need weird Inputs to make sure Check can handle unrelated code without extra violations.

Reply:
This must have misled you. Sorry about that.
> Input for NoLineWrap should check line wraps only.

To be clear, this input is in `src/it` and in the package `com.google.checkstyle.test.chapter3filestructure.rule332nolinewrap`. It is used for IT for google checks.

There's a dedicated `InputNoLineWrapGood.java` in `src/test` for UT.

Reply:
I just tested your code, without this change ... it is not required.
Please do not touch unrelated Inputs, they should stay in most weird way - it is by design.

Reply:
I can't recall why `InputNoLineWrapGood.java` was changed. Re-tested and it looks ok after reverting.  Reverted.